src/Rings/mpoly-localizations.jl

```diff
@@ -1581,12 +1592,12 @@ LocalizedBiPolyArray(
 LocalizedBiPolyArray(
     W::MPolyLocalizedRing{BRT, BRET, RT, RET, MST}, 
     I::MPolyIdeal{RET}
-  ) where {BRT, BRET, RT, RET, MST} = LocalizedBiPolyArray(W, W.(gens(I)), default_shift(W), default_ordering(W))
+  ) where {BRT, BRET, RT, RET, MST} = LocalizedBiPolyArray(W, W.(gens(I)), default_shift(W), default_ordering(W).o)
```

Comment:
I think we should use a getter-method here. What is `.o`?

Reply:
I don't think there is one for this field (it probably should be in `orderings.jl`).

Reply:
We should ask for it. @wbhart Can you help us out here?